packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1091,9 +1110,16 @@ export default class SalesforceCRMService implements CRM {
           );
           if (dateValue) {
             writeOnRecordBody[field.name] = dateValue;
+            continue;
           }
         }
       }
+
+      log.error(
+        `No value found for field ${field.name} with value ${
+          personRecord[field.name]
+        }, field config ${JSON.stringify(fieldConfig)} and Salesforce config ${JSON.stringify(field)}`
+      );
```

Comment:
Additional error logging when a field isn't being written to. The Salesforce field config doesn't contain any PII. The only *possible* PII is the field value but we're already logging that.